oci-unit-tests/standalone-checks_test.sh

```diff
@@ -93,9 +110,151 @@ test_all_supported_architectures_are_available()
         done
     done
 
-    rm -rf "${utilsdir}"
-
     assertTrue "Not all supported architectures are available" "$ret"
 }
 
+test_images_hashes_are_equal()
+{
+    # We currently support only the "ubuntu" namespace, because it's
+    # the only one that is published on both registries.
+    local namespaces="ubuntu"
+    local images_list
+    local ret=0
+
+    if [ -z "${DOCKER_USERNAME}" ] || [ -z "${DOCKER_PASSWORD}" ]; then
+        echo "E: You must specify the DOCKER_USERNAME and DOCKER_PASSWORD environment variables." > /dev/stderr
+        exit 1
+    fi
+    export DOCKER_USERNAME DOCKER_REGISTRY
+
+    for namespace in ${namespaces}; do
+        # Verify that all supported registries have the same list of
+        # images available.
+        local images_list
+
+        images_list=""
+        for registry in ${SUPPORTED_REGISTRIES}; do
+            if [ -z "${images_list}" ]; then
+                images_list=$("${UTILSDIR}"/list-all-images.sh \
+                                           --registry "${registry}" \
+                                           --namespace "${namespace}" | grep -Fvw ubuntu | sort)
+            else
+                if [ "${images_list}" != "$("${UTILSDIR}"/list-all-images.sh \
+                                                         --registry "${registry}" \
+                                                         --namespace "${namespace}" | grep -Fvw ubuntu | sort)" ]; then
+                    echo "E: Registry '${registry}' differs in the list of available images. Aborting." > /dev/stderr
```

Comment:
Yeah, I thought about doing that but then got discouraged because it will require some massaging of the data (one of the downsides of using shell script for these tests).  I'll see what I can do.